_includes/scenarios/integrate/create-attribute-for-peer.md

```diff
@@ -1,15 +1,237 @@
-{% include warnings/documentation-is-prerelease %}
+There are many situations in which an Identity wants to create an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) or a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) for another Identity, for example:
 
-# Flow
+- A university wants to send a graduate their degree certificate.
+- A company wants to provide an employee with their business email address at the start of their employment.
 
-It is possible for an Identity to create an Attribute for another Identity. To achieve this, a Request must be sent to the peer Identity with respective RequestItems. The peer identity must manually accept the Request. Options to create Attributes:
+An Identity has several options for requesting an Attribute creation. This guide covers how an Identity can request the creation of an Attribute for a peer so that the [Attribute Value]({% link _docs_integrate/attribute-values.md %}) is only set by the Identity itself and cannot be modified by the peer when accepting the Request.
 
-- A CreateAttributeRequestItem to create a given Attribute that the peer can accept and thus an attribute is created. The peer is not able to change/overrule this Attribute value. This is great for reusable information which shouldn't be changed by the Identity, e.g. a certificate.
-- A ProposeAttributeRequestItem to propose a given Attribute to the peer. The peer is able to overrule this Attribute and instead select a different Attribute to use. This comes in handy if one knows personal information of a peer, but does not know if this personal information is still valid.
+If the peer should be able to adjust the Attribute offered for creation, the Propose attribute to peer guide must be consulted instead.
```

Comment:
Also, mention the Read attribute scenario here, if the Recipient should be able to completely configure the Attribute itself.